### ISSUE TITLE
Chrome Errors in project preview "TypeError Could not fetch dependencies, please try again in a couple seconds"

### ISSUE THREAD

**Issue by akutruff** [codesandbox/codesandbox-client#6825]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

Under Chrome, the sandbox located here produces an error in the embedded browser:

https://codesandbox.io/s/github/akutruff/dimmer/tree/master/apps/cra

Firefox works fine.

```
TypeError
Could not fetch dependencies, please try again in a couple seconds: Cannot read properties of null (reading 'match')
s
https://codesandbox.io/static/js/sandbox.77d00c338.js:1:138474
(anonymous function)
https://codesandbox.io/static/js/sandbox.77d00c338.js:1:170761
i
https://codesandbox.io/static/js/sandbox.77d00c338.js:1:167040

```

## How has this issue affected you? What are you trying to accomplish?

Project broken in chrome.

### To Reproduce

Go to sandbox here: https://codesandbox.io/s/github/akutruff/dimmer/tree/master/apps/cra and try to see the app.
If you copy the embedded preview url into a new browser tab, the error will also occur.  If you "empty cache and hard reload" on the tab with the preview url, it will actually work *once*.  If you refresh that page again, the error occurs again.  "empty cache and hard reload" on the full sandbox does not fix the problem.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |       latest       |
| Browser          |      Chrome        |
| Operating System |      Windows        |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by CompuIves** [comment]
Hey! I cannot seem to repro this issue, are you still experiencing this? 
<img width="1912" alt="image" src="https://user-images.githubusercontent.com/587016/184703036-815be044-5a76-4b95-8401-160f9d993ac5.png">

I've seen before that an adblocker could prevent some dependencies from being downloaded. Could that maybe be it?

**Comment by akutruff** [comment]
Yeah, unfortunately.  It's super weird.  Perhaps try refreshing the page a lot, or opening it other browsers?   It worked fine until I I set new NPM versions for the dependencies.

**Comment by akutruff** [comment]
![image](https://user-images.githubusercontent.com/874049/184704977-647989a9-cc5d-4380-ad9c-d40b5b49e2e8.png)

**Comment by wilsoft-gt** [comment]
I can confirm the exact same issue, but in my case it takes a long time to install the dependencies, sometimes it will throw the same error and sometimes it will render the app but that is when it takes a long time to install them.
![image](https://user-images.githubusercontent.com/27575726/185031157-d1c71f45-d49d-40a2-8019-e25201198b97.png)

And after it install babel it will throw this error
![image](https://user-images.githubusercontent.com/27575726/185030929-db996e2b-7730-4145-af77-8acdf2ab6a7a.png)

No matter which version of NPM I choose, it takes a long time to compile a project.

Ubuntu 22.04
Tried on all my browsers edge, chrome, firefox and the issue is the same.

**Comment by JamesACS** [comment]
Hey @wilsoft-gt - I've just opened the URL in your screenshot and it's loading perfectly. 

Out of interest, do you have a device with a different operating system that you could try it on?

**Comment by akutruff** [comment]
@JamesACS I disabled all ad blocking and its still happening for me.  

Now, I'm getting CORS errors when referencing my packages as well.

![image](https://user-images.githubusercontent.com/874049/185644397-f01c4440-d15d-4abb-a2de-360fcb613702.png)

**Comment by JamesACS** [comment]
It's still loading fine for us @akutruff , are you using incognito mode? 

As above, have you also tried to load it using a browser on a different operating system?

**Comment by wilsoft-gt** [comment]
> Hey @wilsoft-gt - I've just opened the URL in your screenshot and it's loading perfectly.
> 
> Out of interest, do you have a device with a different operating system that you could try it on?

Yes, it is working fine in other OS, it seems that the issue is somewhere in ubuntu but can't find anything related to this anywhere. I check the network and restarted the page and this is what I'm getting, not sure if this is related to the container and ubuntu 22.04 or what. Half a minute trying to fetch babel core and babel runtime and other dependencies are marked as red as well.

![image](https://user-images.githubusercontent.com/27575726/185763200-e4bb563e-5afa-462b-9861-8d554ccfbd80.png)

**Comment by akutruff** [comment]
@JamesACS 

Nope, normal mode.  I've tried all permutations, and it's still failing.

**Comment by akutruff** [comment]
@JamesACS updated link to sandbox

**Comment by baburay23** [comment]
Hi, I have a similar issue and I am using a mac.  Link to the code sandbox is here [https://codesandbox.io/s/8te0rv?module=/example.js&file=/package.json]
I updated the package version to Newskit 6.3.0 as shown in the sandbox, originally the version was 'unstable' it worked fine with 'unstable' as soon as I changed the version I got the same error of not being able to fetch. I tried it incognito and on safari (I am currently using chrome) and it does not work. I was wondering if anyone has any advice?
![Screenshot 2022-10-20 at 13 19 17](https://user-images.githubusercontent.com/54642337/196946716-4619a130-7fa4-4b86-9f33-965d48ff2334.png)

**Comment by landabaso** [comment]
I'm experiencing the same issue and I see from previous discussions that this has been a persistent problem for at least a year. @baburay23  and @akutruff , were either of you able to discover a workaround or potential solution?

**Comment by landabaso** [comment]
Ok, I managed to make it work. In my case I was trying to embed a github project. I read[ from other users that adding a tag](https://github.com/codesandbox/codesandbox-client/issues/6443#issuecomment-1115103020) could fix it. And, it worked...
